docs/pages/brownfield/installing-expo-modules.mdx

```diff
@@ -0,0 +1,193 @@
+--- a/android/gradle.properties
++++ b/android/gradle.properties
+@@ -20,4 +20,8 @@ kotlin.code.style=official
++newArchEnabled=false
++
++hermesEnabled=true`}
+  />
+</Step>
+
+<Step label="2">
+  Add the following to the `setting.gradle` file in the `android` directory:
+  <DiffBlock
+    raw={`diff --git a/android/settings.gradle b/android/settings.gradle
+index a26c7c6..dd8802a 100644
+--- a/android/settings.gradle
++++ b/android/settings.gradle
+@@ -1,5 +1,11 @@
++apply from: new File(["node", "--print", "require.resolve('expo/package.json')"].execute(null, rootDir).text.trim(), "../scripts/autolinking.gradle");
++useExpoModules()
++
+ include ':app'
+ includeBuild("../node_modules/@react-native/gradle-plugin")`}
+  />
+</Step>
+
+<Step label="3">
+  From the `android` directory, run the following command:
+  <Terminal cmd={['./gradlew clean']} />
+  Once this completes, run the following command:
+  <Terminal cmd={['./gradlew assembleDebug']} />
+</Step>
+
+<Step label="4">
+  **(Optional)** Complete the following steps if you would like to use [lifecycle listener](/modules/android-lifecycle-listeners/)
+  in your app:
+<Step label="1">
+If you already have a class that extends the `Application` class you can move to step 3.
+If you do not have it, we need to create one. Add a file called `MainApplication.kt` file to your `android/app/src/main/java/com/<your-app-package>` directory with the following content:
+
+<DiffBlock raw={`diff --git a/android/app/src/main/java/com/<my-app-package>/MainApplication.kt b/android/app/src/main/java/com/<my-app-package>/MainApplication.kt
+new file mode 100644
+index 0000000..2c8525a
+--- /dev/null
++++ b/android/app/src/main/java/com/<my-app-package>/MainApplication.kt
+@@ -0,0 +1,19 @@
++package <my.app.package>
++
++import android.app.Application
++import android.content.res.Configuration
++import com.facebook.soloader.SoLoader
++import expo.modules.ApplicationLifecycleDispatcher
++
++class MainApplication() : Application() {
++    override fun onCreate() {
++        super.onCreate()
++        ApplicationLifecycleDispatcher.onApplicationCreate(this)
++    }
++
++    override fun onConfigurationChanged(newConfig: Configuration) {
++        super.onConfigurationChanged(newConfig)
++        ApplicationLifecycleDispatcher.onConfigurationChanged(this, newConfig)
++    }
++}`} />
+</Step>
+<Step label="2">
+Register the class in the `AndroidManifest.xml` file.
+<DiffBlock raw={`diff --git a/android/app/src/main/AndroidManifest.xml b/android/app/src/main/AndroidManifest.xml
+index a71b5e7..d6d406e 100644
+--- a/android/app/src/main/AndroidManifest.xml
++++ b/android/app/src/main/AndroidManifest.xml
+@@ -7,6 +7,7 @@
+     <application
+         android:allowBackup="true"
++        android:name=".MainApplication"
+         android:fullBackupContent="@xml/backup_rules"`} />
+</Step>
+
+<Step label="3">
+If you have your own class extending `Application`, you can add the following to it:
```

Comment:
Explain why you'd want to add it. The phrasing ("you can") says that you have the choice but doesn't explain why you'd want to make that choice or how to decide.